manage.py

```diff
@@ -562,25 +563,84 @@ def _run_field_lists(name, table, tables_only):
         no_field_list_type = sql.SQL('TABLE' if table.is_table else 'VIEW')
         final_summary_type = sql.SQL('VIEW')
 
+    counts_per_path_select = """
+        SELECT
+            {primary_keys},
+            path,
+            GREATEST(sum(array_item), sum(object_property)) path_count
+        FROM
+            {summary_table}
+        CROSS JOIN
+            flatten({summary_table}.{data_column})
+        GROUP BY
+            {primary_keys}, path
+    """
+
+    # for contracts and award add paths across the awardID join.
+    if summary_table in ('contracts_summary', 'awards_summary'):
```

Comment:
It'd be useful to have a test for this section, as I'm not 100% clear on what the difference in the output will be.

Reply:
Tests have beed added which hopefully show the difference.  It required modifying the test data as the awardID/contract link did not exist in the current data.